### DISCUSSION TITLE
Not able to install vendure

### DISCUSSION THREAD

**Discussion by ivsaneesh** [vendurehq/vendure#1708] [Q&A]
Node version using is v16.13.2
npm version 8.15.0

Generating app scaffold
Initializing server
npm ERR! code ENOMEM
npm ERR! syscall spawn
npm ERR! errno -12
npm ERR! spawn ENOMEM
npm verb exit -12
npm timing npm Completed in 72914ms
npm verb unfinished npm timer command:install 1659939248908
npm verb unfinished npm timer reify 1659939248912
npm verb unfinished npm timer reify:build 1659939321279
npm verb unfinished npm timer build 1659939321285
npm verb unfinished npm timer build:deps 1659939321286
npm verb unfinished npm timer build:run:install 1659939321558
npm verb unfinished npm timer build:run:install:node_modules/bcrypt 1659939321558
  ✖ Installing dependencies
    → An error occurred when installing dependencies.
    Generating app scaffold
    Initializing server
{"message":"An error occurred when installing dependencies.","command":"npm install --save --save-exact --loglevel error @vendure/core @vendure/email-plugin @vendure/asset-server-plugin @vendure/admin-ui-plugin mysql typescript@4.3.5 --verbose","context":{}}

**Comment by michaelbromley** [comment]
Looks like you are out of memory. How much ram does your system have?

- https://github.com/npm/npm/issues/11155

**Reply by ivsaneesh** [reply]
Thanks
